From c799e23008994913c72a4cc2f566497a44b03fe6 Mon Sep 17 00:00:00 2001
From: honjow <honjow311@gmail.com>
Date: Tue, 29 Jul 2025 11:06:38 +0800
Subject: [PATCH] platform/x86: msi-wmi-platform: update tdp range

---
 drivers/platform/x86/msi-wmi-platform.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/platform/x86/msi-wmi-platform.c b/drivers/platform/x86/msi-wmi-platform.c
index 896fb11af696..bf077a63f70a 100644
--- a/drivers/platform/x86/msi-wmi-platform.c
+++ b/drivers/platform/x86/msi-wmi-platform.c
@@ -234,7 +234,7 @@ static struct msi_wmi_platform_quirk quirk_gen1 = {
 	.charge_threshold = true,
 	.dual_fans = true,
 	.restore_curves = true,
-	.ppt_min = 8,
+	.ppt_min = 5,
 	.ppt_pl1_spl_max = 43,
 	.ppt_pl2_sppt_max = 45
 };
@@ -243,7 +243,7 @@ static struct msi_wmi_platform_quirk quirk_gen2 = {
 	.charge_threshold = true,
 	.dual_fans = true,
 	.restore_curves = true,
-	.ppt_min = 8,
+	.ppt_min = 5,
 	.ppt_pl1_spl_max = 30,
 	.ppt_pl2_sppt_max = 37,
 };
-- 
2.39.1

