diff --git a/drivers/platform/x86/oxpec.c b/drivers/platform/x86/oxpec.c
index 51845aae8a44..3a5b02c6baa4 100644
--- a/drivers/platform/x86/oxpec.c
+++ b/drivers/platform/x86/oxpec.c
@@ -44,12 +44,14 @@ static bool unlock_global_acpi_lock(void)
 enum oxp_board {
 	aok_zoe_a1 = 1,
 	aya_neo_2,
+	aya_neo_2s,
 	aya_neo_air,
 	aya_neo_air_1s,
 	aya_neo_air_plus_mendo,
 	aya_neo_air_pro,
 	aya_neo_flip,
 	aya_neo_geek,
+	aya_neo_geek_1s,
 	aya_neo_kun,
 	orange_pi_neo,
 	oxp_2,
@@ -144,6 +146,13 @@ static const struct dmi_system_id dmi_table[] = {
 			DMI_MATCH(DMI_BOARD_NAME, "AYANEO 2"),
 		},
 		.driver_data = (void *)aya_neo_2,
+	},
+		{
+		.matches = {
+			DMI_MATCH(DMI_BOARD_VENDOR, "AYANEO"),
+			DMI_MATCH(DMI_BOARD_NAME, "AYANEO 2S"),
+		},
+		.driver_data = (void *)aya_neo_2s,
 	},
 	{
 		.matches = {
@@ -187,6 +196,13 @@ static const struct dmi_system_id dmi_table[] = {
 		},
 		.driver_data = (void *)aya_neo_geek,
 	},
+	{
+		.matches = {
+			DMI_MATCH(DMI_BOARD_VENDOR, "AYANEO"),
+			DMI_MATCH(DMI_BOARD_NAME, "GEEK 1S"),
+		},
+		.driver_data = (void *)aya_neo_geek_1s,
+	},
 	{
 		.matches = {
 			DMI_MATCH(DMI_BOARD_VENDOR, "AYANEO"),
@@ -716,11 +732,13 @@ static int oxp_pwm_enable(void)
 		return write_to_ec(ORANGEPI_SENSOR_PWM_ENABLE_REG, PWM_MODE_MANUAL);
 	case aok_zoe_a1:
 	case aya_neo_2:
+	case aya_neo_2s:
 	case aya_neo_air:
 	case aya_neo_air_plus_mendo:
 	case aya_neo_air_pro:
 	case aya_neo_flip:
 	case aya_neo_geek:
+	case aya_neo_geek_1s:
 	case aya_neo_kun:
 	case oxp_2:
 	case oxp_fly:
@@ -741,12 +759,14 @@ static int oxp_pwm_disable(void)
 		return write_to_ec(ORANGEPI_SENSOR_PWM_ENABLE_REG, PWM_MODE_AUTO);
 	case aok_zoe_a1:
 	case aya_neo_2:
+	case aya_neo_2s:
 	case aya_neo_air:
 	case aya_neo_air_1s:
 	case aya_neo_air_plus_mendo:
 	case aya_neo_air_pro:
 	case aya_neo_flip:
 	case aya_neo_geek:
+	case aya_neo_geek_1s:
 	case aya_neo_kun:
 	case oxp_2:
 	case oxp_fly:
@@ -767,12 +787,14 @@ static int oxp_pwm_read(long *val)
 		return read_from_ec(ORANGEPI_SENSOR_PWM_ENABLE_REG, 1, val);
 	case aok_zoe_a1:
 	case aya_neo_2:
+	case aya_neo_2s:
 	case aya_neo_air:
 	case aya_neo_air_1s:
 	case aya_neo_air_plus_mendo:
 	case aya_neo_air_pro:
 	case aya_neo_flip:
 	case aya_neo_geek:
+	case aya_neo_geek_1s:
 	case aya_neo_kun:
 	case oxp_2:
 	case oxp_fly:
@@ -811,12 +833,14 @@ static int oxp_pwm_fan_speed(long *val)
 		return read_from_ec(OXP_2_SENSOR_FAN_REG, 2, val);
 	case aok_zoe_a1:
 	case aya_neo_2:
+	case aya_neo_2s:
 	case aya_neo_air:
 	case aya_neo_air_1s:
 	case aya_neo_air_plus_mendo:
 	case aya_neo_air_pro:
 	case aya_neo_flip:
 	case aya_neo_geek:
+	case aya_neo_geek_1s:
 	case aya_neo_kun:
 	case oxp_fly:
 	case oxp_mini_amd:
@@ -844,12 +868,14 @@ static int oxp_pwm_input_write(long val)
 		val = (val * 184) / 255;
 		return write_to_ec(OXP_SENSOR_PWM_REG, val);
 	case aya_neo_2:
+	case aya_neo_2s:
 	case aya_neo_air:
 	case aya_neo_air_1s:
 	case aya_neo_air_plus_mendo:
 	case aya_neo_air_pro:
 	case aya_neo_flip:
 	case aya_neo_geek:
+	case aya_neo_geek_1s:
 	case aya_neo_kun:
 	case oxp_mini_amd:
 	case oxp_mini_amd_a07:
@@ -886,12 +912,14 @@ static int oxp_pwm_input_read(long *val)
 		*val = (*val * 255) / 184;
 		break;
 	case aya_neo_2:
+	case aya_neo_2s:
 	case aya_neo_air:
 	case aya_neo_air_1s:
 	case aya_neo_air_plus_mendo:
 	case aya_neo_air_pro:
 	case aya_neo_flip:
 	case aya_neo_geek:
+	case aya_neo_geek_1s:
 	case aya_neo_kun:
 	case oxp_mini_amd:
 	case oxp_mini_amd_a07:
