diff --git a/sound/pci/hda/cs35l41_hda_property.c b/sound/pci/hda/cs35l41_hda_property.c
index e9dd5b94e70b..b7655678bb30 100644
--- a/sound/pci/hda/cs35l41_hda_property.c
+++ b/sound/pci/hda/cs35l41_hda_property.c
@@ -83,7 +83,7 @@ static int asus_rog_2023_ally_fix(struct cs35l41_hda *cs35l41, struct device *ph
 	const char *rog_ally_bios_num = rog_ally_bios_ver + 6; // Dropping the RC71L. part before the number
 	int rog_ally_bios_int;
 	kstrtoint(rog_ally_bios_num, 10, &rog_ally_bios_int);
-	if(rog_ally_bios_int >= 330){
+	if(rog_ally_bios_int >= 9330){
 		printk(KERN_INFO "DSD properties exist in the %d BIOS\n", rog_ally_bios_int);
 		return -ENOENT; //Patch not necessary. Exiting...
 	}
